### ISSUE TITLE
bettercap not working on wsl ubuntu 

### ISSUE THREAD

**Issue by funnimonkedev** [bettercap/bettercap#1028]
here is the issue it is leaving me:     
    /var/lib/gems/3.0.0/gems/bettercap-1.6.2/lib/bettercap/shell.rb:25:in ``'
    /var/lib/gems/3.0.0/gems/bettercap-1.6.2/lib/bettercap/shell.rb:25:in `block in execute'
    /var/lib/gems/3.0.0/gems/bettercap-1.6.2/lib/bettercap/shell.rb:23:in `times'
    /var/lib/gems/3.0.0/gems/bettercap-1.6.2/lib/bettercap/shell.rb:23:in `execute'
    /var/lib/gems/3.0.0/gems/bettercap-1.6.2/lib/bettercap/network/network.rb:20:in `get_gateway'
    /var/lib/gems/3.0.0/gems/bettercap-1.6.2/lib/bettercap/context.rb:117:in `update!'
    /var/lib/gems/3.0.0/gems/bettercap-1.6.2/lib/bettercap/options/options.rb:68:in `parse!'
    /var/lib/gems/3.0.0/gems/bettercap-1.6.2/bin/bettercap:30:in `<top (required)>'
    /usr/local/bin/bettercap:25:in `load'
    /usr/local/bin/bettercap:25:in `<main>'
![image](https://user-images.githubusercontent.com/113639481/234426847-5928c34d-24f9-49f8-b0e1-49b05f5affd6.png)

**Comment by diehard88** [comment]
bettercap 1.6.2?

the latest version is 2.32.0.

are you using wsl 2?

<img width="660" alt="bettercap wsl2" src="https://github.com/bettercap/bettercap/assets/25366502/5e2887ce-2eca-4935-a25f-c86cefe844b1">

**Comment by serenityelise** [comment]
I had the same issue. I had to do the following in Ubuntu:

1. Update the package index:  # sudo apt-get update
2. Install bettercap deb package: # sudo apt-get install bettercap

Now when I # sudo bettercap --help, it seems to all load up as it should, and just an FYI, I am a complete newb at all of this. I had to google it 

this is the link I went to for Ubuntu: https://ubuntu.pkgs.org/22.04/ubuntu-universe-amd64/bettercap_2.32.0-1_amd64.deb.html
